### ISSUE TITLE
all-lectures.tex does not compile with lecturehandouts.cls on ubuntu 22.04

### ISSUE THREAD

**Issue by bjornregnell** [lunduniversity/introprog#649]
what is the diff in packages between lecturenotes.cls nd lecturehandouts.cls

this is the error:
```
(./all-lectures.vrb
Runaway argument?
\relax \beamer@origitem {\ifhmode \unskip \unskip 
! Paragraph ended before \@car was complete.
<to be read again> 
                   \par 
l.7     \item
              Kursens \Alert{slutna} sida bakom inloggningsvägg: \url{https...
```
Seems to be casued by `\item[]` for non-bullet items (this works with lecturenotes)

**Comment by bjornregnell** [comment]
This problem occurs on Ubuntu 22.04 but not 20.04 or earlier.

**Comment by bjornregnell** [comment]
Some package missing? Or some package versions that are old?

**Comment by bjornregnell** [comment]
fixed by 9cb0e052d6f653fd657a57607144bb3e6d23a063 and ec8e664e59eb7f266f2106dddb1eb5355eb7fdd7
